README.md

```diff
@@ -173,6 +174,21 @@ Generating a new Rails application using Hyrax's template above takes cares of a
 * Loading all of Hyrax's database migrations into your application's database
 * Loading Hyrax's default workflows into your application's database
 
+## Message Queue
+Many of the services performed by Hyrax are resource intensive, and therefore are well suited to running as background jobs that can be managed and executed by a Message Queue system. Examples include file ingest, derivative generation, characterization and Solr indexing. Hyrax implements these jobs using [ActiveJob](http://edgeguides.rubyonrails.org/active_job_basics.html), allowing you to choose the message queue system of your choice.
```

Comment:
Can you move the examples to bullet point. I believe that helps with scannability.